### ISSUE TITLE
bug with the track selection and time period

### ISSUE THREAD

**Issue by kgomes** [kgomes/odss#44]
_From [monique....@gmail.com](https://code.google.com/u/100912901724892609704/) on October 20, 2010 14:19:09_

yeahhh I found an actual bug!! What steps will reproduce the problem? 1. Erase all tracks (more visible), and select time window = 3 day
2. Show one track (eg. Tethys)
3. Change time window (eg. 6h) What is the expected output? What do you see instead? 1. The track doesn't change (still shows the 3 day track)
2. Unticking "show platform track" for Tethys doesn't erase the track.
In conclusion seems like the track is frozen. The only way I've found to erase it is reload the page. Please use labels and text to provide additional information. (from http://dss.mbari.org/DSS/ )

_Original issue: http://code.google.com/p/mbari-dss/issues/detail?id=46_

**Comment by kgomes** [comment]
_From [kgo...@gmail.com](https://code.google.com/u/106027714978714309234/) on November 20, 2012 09:41:51_

This was in old version of the ODSS.

**Status:** WontFix
